chcemvediet/tests/__init__.py

```diff
@@ -31,6 +32,11 @@ class CustomTestRunner(DiscoverRunner):
     def setup_test_environment(self, **kwargs):
         super(CustomTestRunner, self).setup_test_environment(**kwargs)
         settings.LANGUAGE_CODE = u'en'
+        os.environ[u'RECAPTCHA_TESTING'] = u'True'
```

Comment:
@viliambalaz Dopln info aj do komentaru v `CustomTestRunner` vyssie. Vzdy ked nieco menis, tak sa pozri dookola, ci to s niecim nesuvisi a uprav aj to.

Reply:
:ok: